examples/EvaluationDataToApplicationInsights/appsettings.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ApplicationInsights": {
+    "ConnectionString": "<connection-string>"
+  },
+  "FeatureManagement": {
+    "FeatureA": {
+      "TelemetryEnabled": true,
+      "EnabledFor": [
+        {
+          "Name": "Targeting",
+          "Parameters": {
+            "Audience": {
+              "Users": [
+                "ross"
+              ],
+              "Groups": [],
+              "DefaultRolloutPercentage": 50
+            }
+          }
+        }
+      ]
+    },
+    "FeatureB": {
+      "TelemetryEnabled": true,
+      "Label": "A Label",
```

Comment:
Shouldn't this be in TelemetryMetadata?

Reply:
Yes. Nothing was being emitted for these fields after the TelemetryMetadata change. Fixed.